front-end/LoanHome/LoanHomeStyle.js

```diff
@@ -0,0 +1,43 @@
+import { StyleSheet } from 'react-native'
+
+const style = StyleSheet.create({
+  loanHomeScroll: {
+    paddingHorizontal: 10,
+    marginTop: 75,
+    flex: 1,
+    zIndex: 1,
+    elevation: 1
+  },
+  navigator: {
```

Comment:
@JialinWang1 sam goes for navigator, this will actually display iOS and Android headers differently which is fine:
https://docs.nativebase.io/Components.html#header-icon-headref